### DISCUSSION TITLE
유니티에 제페토 패키지 파일이 설치가 되지 않습니다.

### DISCUSSION THREAD

**Discussion by jackhookang** [naverz/zepeto-studio-kor#781] [5. Q&A]
![유니티 에러](https://user-images.githubusercontent.com/106004742/169680233-ffb5a0bf-78bb-4b41-844f-000789f82061.png)
유니티에 제페토 패키지 파일이 설치가 되지 않습니다.
설치하면 위와 같은 에러 메세지가 발생하는데 혹시 해결 방법을 아시는 분 계실까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 패키지 관련하여 일시적인 장애가 있었고 복구 완료하였습니다.
감사합니다.

**Reply by jackhookang** [reply]
여전히 이 에러가 해결이 되지 않습니다. 제 데스크탑 PC는 정상적으로 설치가 되는데, 노트북이 정상적으로 설치되지 않습니다.
노트북은 갤럭시북 프로 360 이고, 윈도우 11 입니다.

2020.3.9f1 버전을 통해 Assets -> Import package -> Custom package를 통해 설치를 진행했고  ![image](https://user-images.githubusercontent.com/106004742/170072683-e42ffddb-e6b5-400b-8ae0-58a2872a4585.png)
특이한 점은 Completed! 라는 문구가 뜨지 않습니다.
![image](https://user-images.githubusercontent.com/106004742/170074139-7d7a2648-2fb7-48a9-a737-b289e68ff82d.png)

![image](https://user-images.githubusercontent.com/106004742/170072821-bb3dc72f-c83f-4f2c-95ea-4f838a654d06.png)
그리고 설치된 제페토 템플릿으로 들어가면 위와 같은 에러 창이 뜨고요.

**Reply by Zepeto-TechEvangelist** [reply]
상세한 설명 감사 드립니다. 다른 문제가 없는지 한번 더 확인해 보겠습니다.
